src/app.ts

```diff
@@ -15,6 +15,12 @@ import { Logger } from 'pino';
 
 //import { assert } from 'node:console';
 
+interface Headers {
+  'Content-Length': number;
+  'Content-Type': string;
+  'X-AURORA-ERROR-CODE'?: string;
```

Comment:
While headers are case-insensitive, let's do proper capitalization here, no need for uppercase